src/diffusers/loaders.py

```diff
@@ -322,6 +322,10 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
         attn_processors = {k: v.to(device=self.device, dtype=self.dtype) for k, v in attn_processors.items()}
 
         # set layers
+        print(
+            "All processors are of type: LoRAAttnAddedKVProcessor: ",
+            all(isinstance(attn_processors[k], LoRAAttnAddedKVProcessor) for k in attn_processors),
+        )
```

Comment:
This function is internally called when `load_lora_weights()` is called. Checking this verifies if the right attention processors where set. The output of this is True.